### PR TITLE
Issue #421 - Added puzzle for ordering and filtering floors. A few fi…

### PR DESCRIPTION
**Pull request by emilianodellacasa**
For issue #421 

- I added puzzles for filtering and ordering floors, using factories;
- I made a few fixes on view name and template;

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#482](https://github.com/timelesslounge/timelessis/issues/482) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#482](https://github.com/timelesslounge/timelessis/issues/482) is assigned to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil), here is [why](https://www.0crat.com/footprint/CF9LG1YCA/9b47a680-b4c5-4996-ade0-120d7b8b0742); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: tests/integration/it_floors_test.py [hunk 1]
```diff
@@ -16,6 +15,32 @@ def test_list(client, db_session):
...
+                    <article class=\"floor\">
+                    <header>
+                        <div>
+                        <h1>A</h1>
+    """) == 1
+    assert response.status_code != HTTPStatus.OK
+
+
+@pytest.mark.skip(reason="Order is not yet implemented")
+def test_filtered_list(client, db_session):
```
Comment (amihaiemil) [inline tests/integration/it_floors_test.py]: @emilianodellacasa is the ``db_session`` needed? I guess not

File: tests/integration/it_floors_test.py [hunk 2]
```diff
@@ -16,6 +15,32 @@ def test_list(client, db_session):
...
+    response = client.get(url_for("floor.list", order_by=["description:asc"]))
+    html = response.data.decode('utf-8')
+    assert html.count("""</header>
+
+
+                    <article class=\"floor\">
+                    <header>
+                        <div>
+                        <h1>A</h1>
+    """) == 1
```
Comment (amihaiemil) [inline tests/integration/it_floors_test.py]: @emilianodellacasa indentation here seems strange

File: tests/integration/it_floors_test.py [hunk 3]
```diff
@@ -16,6 +15,32 @@ def test_list(client, db_session):
...
+    assert response.status_code != HTTPStatus.OK
+
+
+@pytest.mark.skip(reason="Order is not yet implemented")
+def test_filtered_list(client, db_session):
+    response = client.get(url_for('floor.list', filter_by=["description=B"]))
+    html = response.data.decode('utf-8')
+    print(html)
+    assert html.count('<h1>A</h1>') == 0
```
[the remark this answers is not in the document]
Comment (emilianodellacasa) [reply tests/integration/it_floors_test.py]: @amihaiemil I am filtering by `description = "B"`, so I expect A to not be shown, does it make sense?

File: tests/integration/it_floors_test.py [hunk 4]
```diff
@@ -16,6 +15,32 @@ def test_list(client, db_session):
...
+                        <h1>A</h1>
+    """) == 1
+    assert response.status_code != HTTPStatus.OK
+
+
+@pytest.mark.skip(reason="Order is not yet implemented")
+def test_filtered_list(client, db_session):
+    response = client.get(url_for('floor.list', filter_by=["description=B"]))
+    html = response.data.decode('utf-8')
+    print(html)
```
[the remark this answers is not in the document]
Comment (emilianodellacasa) [reply tests/integration/it_floors_test.py]: @amihaiemil Ooops, correct!

File: tests/integration/it_floors_test.py (hunk 1 above)
Comment (emilianodellacasa) [reply tests/integration/it_floors_test.py]: @amihaiemil Of course not, thanks

File: tests/integration/it_floors_test.py (hunk 2 above)
Comment (emilianodellacasa) [reply tests/integration/it_floors_test.py]: @amihaiemil I am not sure yet if this is the right way to test it, but I fixed the identation

**Comment by emilianodellacasa** [comment]
@amihaiemil Please check now

File: tests/integration/it_floors_test.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (amihaiemil) [reply tests/integration/it_floors_test.py]: @emilianodellacasa ah, I missed that, thanks. But then can we also add an assertion that ``<h1>B</h1>`` is present?

**Comment by amihaiemil** [comment]
@emilianodellacasa I would just add that assert for B and then it's ok to merge. But however you want, after all :D

**Comment by emilianodellacasa** [comment]
@amihaiemil Done! Take a look and tel me if it's ok to merge :)

**Comment by amihaiemil** [comment]
@emilianodellacasa looks wonderful :D

**Comment by amihaiemil** [comment]
@rultor good to merge

**Comment by emilianodellacasa** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job was finished in 3 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
Job [#482](https://github.com/timelesslounge/timelessis/issues/482) is not in the agenda of @amihaiemil[/z](https://www.0crat.com/u/amihaiemil), can't inspect

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @amihaiemil[/z](https://www.0crat.com/u/amihaiemil), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#482](https://github.com/timelesslounge/timelessis/issues/482) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa)

**Comment by ypshenychka** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Order was finished, quality is "good": +25 point(s) just awarded to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil)

**Comment by 0crat** [comment]
Quality review completed: +6 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**